Solution

Spstv-sys opened the pull request.

Spstv-sys committed:
Solution

vlasiuksasha commented on src/averageAges.js:
```diff
@@ -37,7 +48,21 @@ function calculateMenAverageAge(people, century) {
+    }
+  });
+
+  return count === 0 ? 0 : totalAge / count;
```
Instead of checking if count equals 0 at the end, check at the start if the array is empty and if it is, return 0.

vlasiuksasha commented on src/averageAges.js:
```diff
@@ -15,7 +15,18 @@
+    }
+  });
+
+  return count === 0 ? 0 : totalAge / count;
```
Instead of checking if count equals 0 at the end, check at the start if the array is empty and if it is, return 0.

vlasiuksasha commented on src/averageAges.js:
```diff
@@ -55,7 +80,44 @@ function calculateWomenAverageAge(people, withChildren) {
+
+  const tot = totalAgeDiff / count;
+
+  return count === 0 ? 0 : tot;
```
Instead of checking if count equals 0 at the end, check at the start if the array is empty and if it is, return 0.

vlasiuksasha reviewed (changes requested):
check the comments

Spstv-sys committed:
Update averageAges.js

Instead of checking if count equals 0 at the end, checking at the start if the array is empty and if it is, return 0.
